spec/factories/user.rb

```diff
@@ -1,3 +1,5 @@
 FactoryBot.define do
-  factory :user
+  factory :user do
+    sequence(:uid) { |n| "uid-#{n}" }
```

Comment:
Any reason why you've not gone for a uuid? since that is a more accurate representation of usage.

Reply:
Not really, only that I saw Whitehall first and copied that one. Happy to change it to a UUID.